core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -385,10 +385,13 @@ else
     # display :34.0 would result in starting port 53400 (instead of default 50000 range)
     MY_START_PORT=${DISPLAY/*:/}
     MY_START_PORT=${MY_START_PORT/.*/}
-    MY_START_PORT=`expr 50000 + $MY_START_PORT '*' 100`
+    # Pick a port range starting at 24000 + 200 * display number.
+    # Choose a display number of up to 42 to stay below the
+    # ephemeral port range which typically starts at 32768.
+    MY_START_PORT=`expr 24000 + $MY_START_PORT '*' 200`
   elif [ $MY_START_PORT == "rand" ]; then
-    # pick a random number between 9000 and 49000 that is divisible by 200
-    MY_START_PORT=`expr $RANDOM '%' 200 '*' 200 + 9000`
+    # pick a random number between 12000 and 24000 that is divisible by 200
+    MY_START_PORT=`expr $RANDOM '%' 50 '*' 200 + 12000`
```

Comment:
Looks like you're picking a port between 12000 and 22000 (which is fine). If you want to go up to 24000 you could use 60 instead of 50.